src/hearings/converters/room-id.answer.converter.ts

```diff
@@ -9,7 +9,7 @@ export class RoomIdAnswerConverter implements AnswerConverter {
     return hearingState$.pipe(
       map(state => {
         const hearingResponse = state.hearingRequest.hearingRequestMainModel.hearingResponse;
-        return hearingResponse && hearingResponse.hearingDaySchedule.hearingRoomId;
+        return hearingResponse && hearingResponse.hearingDaySchedule[0].hearingRoomId;
```

Comment:
do we know the hearingdayschedule will be there all the time and if not we may have check hearingDaySchedule.length before pick up the 1st array